### ISSUE TITLE
[BUG] RE-M3 "Mainsail" shoud issues

### ISSUE THREAD

**Issue by omega489** [PorktoberRevolution/ReStocked#683]
**Branch: Develop**

Issue:
Mainsail variant "Boattial (orange/grey)" is identical to the "Boattail (white)" variant. I presume the orange/grey variant is supposed to be orange and grey. 
![image](https://user-images.githubusercontent.com/23028378/67385858-c8d25e80-f58b-11e9-82f2-0a9e1e07d02e.png)

**Comment by Duck1998** [comment]
Confirmed that the bug still exists in _master_ branch.
https://github.com/PorktoberRevolution/ReStocked/blob/7ab6df3bd4cc219769c82a1cc55dfbc4c618dd99/Distribution/Restock/GameData/ReStock/Patches/Engine/restock-engines-liquid-25.cfg#L343-L362
after changing to (Line 360-361)
```
Mainsail_BoattailMeshWhite = false
Mainsail_BoattailMeshOrange = true
```
then it becomes another bug 😂 

![mailsail_boattail](https://user-images.githubusercontent.com/29515995/67409019-87be6680-f5ec-11e9-8bc8-47d0f691bba5.JPG)

**Comment by ChrisAdderley** [comment]
Fixed in ca3293a

**Comment by Poodmund** [comment]
Can confirm that the issue is fixed.
